tox.ini

```diff
@@ -1,4 +1,102 @@
+[tox]
+minversion = 4.0
+skip_missing_interpreters = true
+envlist =
+#    flake8
+#    coverage
+    python{37,38,39}-django32
+    python{38,39,310}-django40
+    python{38,39,310,311}-django41
+    python{38,39,310,311}-django42
+
+toxworkdir = {env:TOX_WORKDIR:.tox}
+
+[gh-actions]
+python =
+    3.7: python37
+    3.8: python38
+    3.9: python39
+    3.10: python310
+    3.11: python311
+
+[testenv]
+setenv =
+    DJANGO_SETTINGS_MODULE=mregsite.settings
+    CI=True
+passenv = MREG_*, GITHUB_*
+deps =
+    -r{toxinidir}/requirements-test.txt
```

Comment:
Add `-r{toxinidir}/requirements.txt`  and only have test depedencies in `requirements-test.txt`.

Reply:
The main downside is that we then have to open up the django version directly in requirements.txt to test against multiple versions. Otherwise we run into dependency conflicts. I opted to keep the version locked for now in prod.